docs/proposals/scheduledjob.md

```diff
@@ -0,0 +1,113 @@
+and time part according to ISO8601 format.
+
+The new `ScheduledJob` object will have the following content:
+
+```go
+// ScheduledJob represents the configuration of a single scheduled job.
+type ScheduledJob struct {
+    TypeMeta
+    ObjectMeta
+
+    // Spec is a structure defining the expected behavior of a job, including the schedule.
+    Spec ScheduledJobSpec
+
+    // Status is a structure describing current status of a job.
+    Status ScheduledJobStatus
+}
+
+// ScheduledJobList is a collection of scheduled jobs.
+type ScheduledJobList struct {
+    TypeMeta
+    ListMeta
+
+    Items []ScheduledJob
+}
+```
+
+`ScheduledJobSpec` structure is defined to contain all the information how the actual
+job execution will look like, including the `JobSpec` from [Job Controller API](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/proposals/job.md)
+and the schedule in ISO8601 format.
+
+```go
+// ScheduledJobSpec describes how the job execution will look like and when it will actually run.
+type ScheduledJobSpec struct {
+
+    // Spec is a structure defining the expected behavior of a job.
+    Spec JobSpec
+
+    // Schedule contains the schedule in ISO8601 format, eg.
+    // - 2015-07-21T14:00:00Z - represents date and time in UTC
+    // - R/2015-07-21T14:00:00Z/P1D - represents endlessly repeating interval (1 day), starting from given date
+    Schedule string
+
+    // SkipOutdated specifies that if AllowConcurrent is false, only the newest job
+    // will be started (default: true), ignoring executions that missed their schedule.
+    SkipOutdated bool
+
+    // BlockOnFailure suspends scheduling of next job runs after a failed one.
+    BlockOnFailure bool
+
+    // AllowConcurrent specified whether concurrent jobs may be started.
+    AllowConcurrent bool
+}
+```
+
+`ScheduledJobStatus` structure is defined to contain some information for scheduled
+job executions (up to a limited amount).  The structure holds objects in three lists:
+* "Pending" list is used as a way to stack executions for job preventing concurrent
+runs, and also to trigger an execution for a job defined without time based scheduling;
+* "Running" list contains all actually running jobs with detailed information;
+* "Failed" list contains information about recently failed jobs.
+
+```go
+// ScheduledJobStatus represents the current state of a Job.
+type ScheduledJobStatus struct {
+    // PendingExecutions are the job runs pending for execution
+    PendingExecutions []JobStatus
+
+    // CurrentExecutions are the job run currently executing
+    RunningExecutions []JobStatus
+
+    // CompletedExecutions tracks previously scheduled jobs (up to a limited amount)
+    CompletedExecutions []JobStatus
+
+    // FailedExecutions tracks previously failed jobs
+    FailedExecutions []JobStatus
+
+    // ScheduleCount tracks the amount of successful executions for this job
```

Comment:
The comment says ScheduleCount but the field is called CompletedCount.

BTW do you really need this field and the next one? The previous fields contain lists of the (presumably recently) completed (successfully or unsuccessfully) jobs. I'm not sure you need anything more than that.